### ISSUE TITLE
New Spec: lsscsi

### ISSUE THREAD

**Issue by xiangce** [RedHatInsights/insights-frontend-assets#30]
https://github.com/RedHatInsights/insights-core/pull/1095
~~~
lsscsi = RegistryPoint()
lsscsi = simple_command("/usr/bin/lsscsi")
lsscsi = simple_file("insights_commands/lsscsi")
~~~

**Comment by kylape** [comment]
Resolved in fb6b10e

**Comment by xiangce** [comment]
Thanks.
